src/app/app.module.ts

```diff
@@ -15,6 +14,7 @@ import { Broadcaster, Logger } from 'ngx-base';
 import { DropdownModule } from 'ngx-dropdown';
 import { AuthenticationService, UserService, HttpService } from 'ngx-login-client';
 import { WidgetsModule } from 'ngx-widgets';
+import { MomentModule } from 'angular2-moment';
```

Comment:
Moment is the underlying time mgmt library. angular2-moment adds pipes for moment.

ATM runtime-console uses angular2-moment, and planner /ngx-widgets has one DIY pipe based on moment. Would suggest we standardise on angular2-moment across the board.